include/tuvx/cross_section.hpp

```diff
@@ -0,0 +1,7 @@
+
+class tuvx_cross_section {
+  private:
+    
+
+
+};
```

Comment:
Is this working in progress? Would it make sense that you submit this when it's complete?

Reply:
I did not edit this file. Not sure how it got added to my PR